KEYLESS.md

```diff
@@ -116,6 +116,31 @@ Signature timestamps are checked in the [rekor](https://github.com/sigstore/reko
 * Probably a lot more: This is very experimental.
 * More OIDC providers: Obvious.
 
+## Public Staging Environment
+
+There is a public staging environment that is running Fulcio, Rekor and OIDC issuer endpoints at the below address:
+
+* https://fulcio.sigstage.dev
+* https://rekor.sigstage.dev
+* https://oauth2.sigstage.dev/auth
+
+These instances are operated and maintained in the same manner as the public production environment for Sigstore.
```

Comment:
There will be no SLO provided nor the same protection of the root key material for TUF. We should loudly note this, that this is staging and never should be used for production.

Reply:
added note about slo and not for production use.